cloudformation-templates/zs_cc_cf_template_simple.yaml

```diff
@@ -259,27 +256,46 @@ Resources:
       SourceDestCheck: 'false'
       Tags:
         - Key: Network
-          Value: !Sub '${AWS::StackName}-Servic3'
+          Value: !Sub '${AWS::StackName}-Service2'
         - Key: Name
-          Value: !Sub '${AWS::StackName}-CloudConnectorSrvcIF3'
+          Value: !Sub '${AWS::StackName}-CloudConnectorSrvcIF2'
   ServiceXface4:
+    Condition: CreateCCMediumServiceXfaces
+    Type: 'AWS::EC2::NetworkInterface'
+    Properties:
+      SubnetId: !Ref CCSubnetID
+      Description: !If
+        - CreateCCMediumServiceXfaces
+        - 'Interface for lb vip'
```

Comment:
the "LB VIP" is technically the ENA1 interface. This last interface is the dedicated "SMLB" interface.

Reply:
I didnt like to put SMLB as SM is more of an internal naming term

Reply:
hmm what about 'Interface for internal LB' or 'CC LB' ?

Reply:
Ok, i'll use internal LB